example-ideal.txt

```diff
@@ -0,0 +1,20 @@
+// Implementation example for empty picobot maze
+
+state leftTraverse {
+	move left;
+	else stay, downTraverse
```

Comment:
Oh, it is implicit. `else downTraverse` works. I guess our ideal example was less ideal than our reality in this instance!